nextcord/client.py

```diff
@@ -1292,6 +1297,92 @@ async def on_ready():
         _log.debug("%s has successfully been registered as an event", coro.__name__)
         return coro
 
+    def add_listener(self, func: CoroFunc, name: str = MISSING) -> None:
+        """The non decorator alternative to :meth:`.listen`.
```

Comment:
I'm not sure as to if these should have versionadded, since they have been added to `Client`.

Reply:
e2724047